diambra/arena/arena_gym.py

```diff
@@ -195,7 +201,7 @@ def __init__(self, env_settings):
         # Define action and observation space
         # They must be gym.spaces objects
 
-        if env_settings["action_space"][0] == "multi_discrete":
+        if env_settings.action_space == "multi_discrete":
```

Comment:
You could make these enums or create a action space type and export on for each option. That way you wouldn't need to check these with conditions here.

Reply:
As in the comment below: noted, this will be included in the rework that will make use of protobuffer class directly